docs/guides/deploying-colony-locally.md

```diff
@@ -0,0 +1,135 @@
+
+Ganache was installed when you set up all the dependencies earlier. You can start a CLI version of it like so (run in a different terminal window from the `colonyNetwork` directory):
+
+```bash
+npm run start:blockchain:client
+```
+
+This will run Ganache on port `8545`. Do not close the window, we're about to deploy the contracts!
+
+:::tip
+Ganache will also create a file called `ganache-accounts.json` in the `colonyNetwork` directory. This file contains the public and private keys of development accounts you can use (see the `private_keys` property at the bottom). These accounts will be used to deploy the ColonyNetwork contracts and will be funded with ETH on the local blockchain. You are encouraged to use them in your own code!
+:::
+
+### Deploying the Colony Network contracts
+
+To deploy the Colony Network contracts to the running development RPC node (Ganache) we use the following command (**make sure you're using the same NodeJS version as for Ganache**):
+
+
+<Tabs>
+<TabItem value="docker" label="Using Docker" default>
+
+```bash
+npx truffle migrate --reset --compile-all
+```
+</TabItem>
+
+<TabItem value="nodocker" label="Without Docker" default>
+
+```bash
+DISABLE_DOCKER=true npx truffle migrate --reset --compile-all
+```
+
+</TabItem>
+</Tabs>
+
+This will run Truffle's so called **migrations**, to deploy all contracts. Keep in mind that this will not only deploy the main `ColonyNetwork` contract, but also set up the MetaColony alongside all its extensions. Please be patient, this will take some time. When you see something akin to the following output, everything was successfully deployed:
+
+```
+Summary
+=======
+> Total deployments:   13
+> Final cost:          0.57571114 ETH
+```
+
+The migration scripts will also create a file called `etherrouter-address.json`. It contains the address for the main entry point for the Colony Contracts and can be instantiated as the `ColonyNetwork` contract. From this one you will be able to figure out all relevant addresses by just calling the corresponding functions on the `ColonyNetwork` contract.
+
+:::info
+Why `etherrouter-address`? Colony uses the so called _EtherRouter_ pattern for upgradeable Smart Contracts. Read more about that [here](https://blog.colony.io/writing-upgradeable-contracts-in-solidity-6743f0eecc88/). Or watch [this video](https://www.youtube.com/watch?v=Sw9O2LWgWC0). It's up to you :)
+:::
+
+What does that mean in practice? Act as if the `etherrouter-address` is the Address for the deployed `ColonyNetwork` contract.
+
+## Talking to the Colony Network
+
+While the Ganache RPC is still running, we can now actually communicate with the deployed contracts. Let's find out the address of the `EtherRouter` and ask for the locally deployed MetaColony's address:
+
+```bash
+cat etherrouter-address.json       
+# {"etherRouterAddress":"0x5CC4a96B08e8C88f2c6FC5772496FeD9666e4D1F"}
```

Comment:
Should these comments be removed?

Reply:
These serve as example answers from the command above. I can try to make that more clear